intel-sgx/dcap-artifact-retrieval/Cargo.toml

```diff
@@ -28,7 +28,7 @@ mbedtls = { version = "0.12.3", features = [
     "std",
 ], default-features = false }
 num_enum = { version = "0.7", features = ["complex-expressions"] }
-pcs = { version = "0.6.0", path = "../pcs" }
```

Comment:
The latest published version of pcs on crates.io is 0.6.0, shouldn't this be 0.6.1?

Reply:
The last version in pcs was `0.6.1` so I just updated the patch version by 1.